core/integration/storagetest/mutation_logs_reader.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2019 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package storagetest
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/google/keytransparency/core/sequencer"
+
+	pb "github.com/google/keytransparency/core/api/v1/keytransparency_go_proto"
+)
+
+// LogsReadWriter supports test's ability to write to and read from the mutation logs.
+type LogsReadWriter interface {
+	sequencer.LogsReader
+	// Send submits the whole group of mutations atomically to a log.
+	// TODO(gbelvin): Create a batch level object to make it clear that this a batch of updates.
```

Comment:
I think the idea was to create a distinct type to hold batches of mutations. 
This would make the ReadLogs API much clearer since we could return []Batches rather than having the semi-awkard current situation of returning more than `batchSize` of mutations. 

@mhutchinson -- your thoughts?

Reply:
If a batch is a first-class object then it seems subtle and indirect that it is constructed by passing in a bunch of `EntryUpdate`s at the same time. This feels like it should be a slice at a minimum.

For the purposes of just this method I wouldn't go beyond turning the varargs `mutation` into a slice, but an API shouldn't be designed a method at a time so I'd need to re-review the whole API & future plans to make a call on the situation.